### ISSUE TITLE
Saving/loading from palette sometimes moves/destroys pins

### ISSUE THREAD

**Issue by Twisol** [PleasingFungus/Silicon-Zeroes#99]
If I save my Jump (Asides) solution to palette and load it into the Delay Jump puzzle, some wire pins get shifted off of where they "should" be, or the pinned points are removed entirely.

* Jump: [puzz-Jump.mftsv.txt](https://github.com/PleasingFungus/Silicon-Zeroes/files/1341728/puzz-Jump.mftsv.txt)
* Palette group: [group-5.mftsv.txt](https://github.com/PleasingFungus/Silicon-Zeroes/files/1341724/group-5.mftsv.txt)
* Delay Jump: [puzz-Delay Jump.mftsv.txt](https://github.com/PleasingFungus/Silicon-Zeroes/files/1341729/puzz-Delay.Jump.mftsv.txt)

## Example

Before saving to palette:
![image](https://user-images.githubusercontent.com/61392/30980730-ebe6a708-a436-11e7-94c1-11fb5b93b756.png)

After loading from palette:
![image](https://user-images.githubusercontent.com/61392/30980751-ff1dc04a-a436-11e7-9a7a-d42ec65315a3.png)

**Comment by PleasingFungus** [comment]
This is about the orange (opcode) and dark red (destination) pins, right?

**Comment by Twisol** [comment]
Yes, that's correct. It only seems to affect wires attached to the op-sel modules.

**Comment by PleasingFungus** [comment]
OK, will take a look when I get a chance. Thanks for reporting!
